packaging/macOS/parsec.spec

```diff
@@ -61,6 +61,10 @@ app = BUNDLE(coll,
             'CFBundleURLName': 'com.scille.parsec',
             'CFBundleURLSchemes': ['parsec']
          }
-      ]
+      ],
+      'LSEnvironment': {
```

Comment:
Should add a comment to explain why this is needed (given it looks strange not to use the user's locale but to force English everywhere)